rentalPrice.js

```diff
@@ -1,33 +1,33 @@
-// age - age of driver
-// licence - number of full years person holds driving licence
-// clazz - class of the car from 1 (smallest) to 5 (largest) that person wishes to rent
-// acc - has s/he caused any accidents within last year
-// acc2 - has s/he participated (but not caused) in any accidents within last year
-// season - if it is high season or not
-function price(age, licence, clazz, acc, acc2, season) {
+// driverAge - age of driver
+// licenceAge - number of full years person holds driving licence
+// carClass - class of the car from 1 (smallest) to 5 (largest) that person wishes to rent
+// causedAccident - has s/he caused any accidents within last year
+// sufferedAccident - has s/he participated (but not caused) in any accidents within last year
+// isHighSeason - if it is high season or not
+function price(driverAge, licenceAge, carClass, causedAccident, sufferedAccident, isHighSeason) {
```

Comment:
Overall, the function is quite long and hard to follow. Maybe it makes sense to separate it out to smaller functions which each do one thing?